### PR TITLE
[MIG] mgmtsystem: Migration to 14.0

### PR DESCRIPTION
**Pull request by dzungtran89**

### CODE REVIEW

File: mgmtsystem/readme/CREDITS.rst [hunk 1]
```diff
@@ -0,0 +1,3 @@
+The development of this module has been financially supported by:
+
+* Camptocamp
```
Comment (dreispt) [inline mgmtsystem/readme/CREDITS.rst]: I'm quite sure Camptocamp was not involved in the original development of this module.
Can you elaborate on this please?

Comment (nilshamerlinck) [reply mgmtsystem/readme/CREDITS.rst:3]: Hi @dreispt 

It has.

@dzungtran89 is working for Trobz that has been commissioned by Camptocamp to work on this migration.

Comment (nilshamerlinck) [reply mgmtsystem/readme/CREDITS.rst]: Hi @dreispt indeed; however it sponsored this migration; ok for you if `s/development/migration/` ?

Comment (dreispt) [reply mgmtsystem/readme/CREDITS.rst]: @nilshamerlinck Yes, it is OK to do that. Please be specific and say that "The Odoo 14 migration of this module was...".
Sorry for nitpicking on this, but the current wording gives the wrong message and is unfair to the original developers and other migration funders.
Thanks.

**Comment by YvanDotet** [comment]
Hello
Is the mgmtsystem app is ready for v 14 ?
Thanks.

**Comment by dreispt** [comment]
@YvanDotet Yes, I think comments were addressed and this is good to go.

**Review by dreispt — APPROVED** [review]

**Pull request merged**